doc/dev/developer_guide/running-tests-locally.rst

```diff
@@ -103,9 +103,10 @@ vstart_runner.py can take the following options -
           to ``/etc/fuse.conf``.
 
 .. note:: If using the kernel client, the user must have the ability to run
-          commands with passwordless sudo access. A failure on the kernel
-          client may crash the host, so it's recommended to use this
-          functionality within a virtual machine.
+          commands with passwordless sudo access.
+
+.. note:: A failure on the kernel client may crash the host, so it's
+          recommended to use this functionality within a virtual machine.
```

Comment:
These changes have been merged in a separate PR - https://github.com/ceph/ceph/pull/45923.